src/core/gui/toolbarMenubar/ToolPageLayer.cpp

```diff
@@ -183,8 +172,8 @@ void ToolPageLayer::updateMenu() {
 
     addSpecialButtonTop();
 
-    int layerCount = lc->getLayerCount();
-    for (int layer = layerCount; layer > 0; layer--) {
+    auto layerCount = lc->getLayerCount();
+    for (auto layer = layerCount; layer > 0; layer--) {
```

Comment:
Shouldn't this be   
```suggestion
    auto layerCount = lc->getLayerCount();
    for (auto layer = layerCount; layer != 0; ) {
        layer--
```

Reply:
Must the order be reversed?

Reply:
No, not this one: the case layer == 0 (i.e. the background) is treated separatly just belonw this chunk of code.